### DISCUSSION TITLE
Battery Conservation Mode

### DISCUSSION THREAD

**Discussion by benoe77** [jrandiny/yoga-slim7-ubuntu#27] [Ideas]
I think this is a very useful function, worth mentioning here, that we can enable battery conservation mode under linux:

ideapad_laptop solution (recommended) #
This solution requires the kernel module ideapad_laptop to be loaded. The solution is also described in the ArchWiki.

Is ideapad_laptop loaded? #
lsmod | grep ideapad_laptop
Find the right path for your model #
ls /sys/bus/platform/drivers/ideapad_acpi
Search for something that looks similar too VPC2004:00. If you have something different, replace it in the commands below.

Read battery conservation mode status #
Value 1 means conservation mode is enabled, and value 0 means it’s currently disabled.

cat /sys/bus/platform/drivers/ideapad_acpi/VPC2004:00/conservation_mode
Enable battery conservation mode #
echo 1 > /sys/bus/platform/drivers/ideapad_acpi/VPC2004:00/conservation_mode
Disable battery conservation mode #
echo 0 > /sys/bus/platform/drivers/ideapad_acpi/VPC2004:00/conservation_mode

from [here](https://tildehacker.com/lenovo-ideapad-battery-conservation-mode-gnu-linux)

**Comment by jrandiny** [comment]
I will verify this and add it to the readme. Thanks

Added
